docs/source/dev/git_notes.rst

```diff
@@ -45,22 +45,27 @@ This should take you to your fork's page.
 
 Then, you want to clone the fork to your machine::
 
-    git clone git@github.com:your-user-name/statsmodels.git statsmodels-yourname
-    cd statsmodels-yourname
-    git remote add upstream git://github.com/statsmodels/statsmodels.git
+    git clone https://github.com/your-user-name/statsmodels
```

Comment:
why do you switch to the https interface? 
I'm using almost only git@github.com